tests/h/session_test.py

```diff
@@ -69,8 +69,28 @@ def test_model_show_sidebar_tutorial(
         assert 'show_sidebar_tutorial' not in preferences
 
 
+def test_profile_userid_unauthenticated(unauthenticated_request):
```

Comment:
Sounds reasonable. I had a quick go at parameterizing the tests based on their target, just to satisfy my own curiosity, and I agree that duplicating the tests feels cleaner.